docs/modules/ROOT/pages/deprecation-notice-cli.adoc

```diff
@@ -0,0 +1,10 @@
+// tag::deprecation-notice-cli[]
+[NOTE]
+.CLI Client is Changing
+====
+
+From 21 August 2023, the xref:clc:ROOT:overview.adoc[Hazelcast Command-Line Client (CLC)] will replace Hazelcast CLI in all Hazelcast {hazelcast-cloud} environments. Hazelcast CLC introduces a new set of commands specially designed for connecting to and managing your {hazelcast-cloud} clusters.
```

Comment:
Let's make it August 31st, give the engg team one more sprint to finish everything